terraform/aws/modules/module_ecs_service/variables.tf

```diff
@@ -253,3 +259,8 @@ variable "lb_allowed_cidrs" {
   description = "List of CIDRs that will be allowed to access the Load Balancer"
   default = []
 }
+variable "dlt_vpc_id" {
+  type = string
+  description = "VPC where Distributed Load testing environment is hosted"
+  default = "vpc-03f843c08b01876d5"
```

Comment:
remove the default from here and supply via component and module declaration

Reply:
removed now